test/verify/check-realms

```diff
@@ -204,15 +204,30 @@ class TestRealms(MachineCase):
         b.wait_present("#dashboard-hosts a[data-address='x0.cockpit.lan']")
         b.logout()
 
-        # Leave the domain
+        # Test domain info (PR #11096), leave the domain
         b.login_and_go("/system")
         b.wait_present("#system-info-domain a")
         b.wait_in_text("#system-info-domain a", "cockpit.lan")
         b.click("#system-info-domain a")
         b.wait_popup("realms-op")
-        b.wait_visible(".realms-op-leave-only-row")
-        b.wait_in_text(".realms-op-leave-only-row", "cockpit.lan")
-        b.click(".realms-op-apply")
+        if m.image != "rhel-7-6-distropkg":
```

Comment:
I _really_ don't like how this rhel-7-6-distropkg image needs to be special cased everywhere...